docs/src/intro-js.md

```diff
@@ -55,6 +55,12 @@ const { webkit } = require('playwright');
 })();
 ```
 
+Save the script above in a filename of your choosing, such as `first_script.js`. Then execute the script from the command-line using Node.js.
+
+```
+npx node first_script.js
```

Comment:
Looks like you don't need npx here, `node first_script.js` is sufficient.

Reply:
I wanted to show good practices but it's not necessary for the example. I'll remove it.